lib/fluent/compat/output.rb

```diff
@@ -111,6 +111,10 @@ module ChunkSizeCompatMixin
       def size
         self.bytesize
       end
+
+      def contains_size
```

Comment:
`size_of_events ` looks good.
fixed it.

https://github.com/fluent/fluentd/pull/1137/commits/8b3f97a2c90680a7d5a8f7820bb33a818c261d0f#diff-ff8e41ed5c655ada05e38acdcee75eb8R115